src/main/java/uk/gov/hmcts/reform/civil/config/properties/notification/NotificationsProperties.java

```diff
@@ -153,4 +153,7 @@ public class NotificationsProperties {
     @NotEmpty
     private String standardDirectionOrderDJTemplate;
 
+    @NotEmpty
+    private String respondentChangeOfAddressForCui;
```

Comment:
Could this field name be more specific and also in line with other fields. could it have a template at the end as it represents the template name isn't it?

Reply:
Field name is changed after discussion